marklogic-data-hub/src/test/java/com/marklogic/hub/master/MasterTest.java

```diff
@@ -97,18 +109,20 @@ private HubModuleManager getPropsMgr() {
     }
 
     @Test
-    public void testMasterStep() throws Exception {
-        installProject();
-
-        getDataHub().clearDatabase(HubConfig.DEFAULT_FINAL_SCHEMAS_DB_NAME);
-        assertEquals(0, getDocCount(HubConfig.DEFAULT_FINAL_SCHEMAS_DB_NAME, "http://marklogic.com/xdmp/tde"));
-
-        getDataHub().clearDatabase(HubConfig.DEFAULT_STAGING_SCHEMAS_DB_NAME);
-        assertEquals(0, getDocCount(HubConfig.DEFAULT_STAGING_SCHEMAS_DB_NAME, "http://marklogic.com/xdmp/tde"));
-
-        installHubArtifacts(getDataHubAdminConfig(), true);
-        installUserModules(getDataHubAdminConfig(), true);
+    public void testMatchEndpoint() throws Exception {
+        Flow flow = flowManager.getFlow("myNewFlow");
+        if (flow == null) {
```

Comment:
This doesn't require a change, but in a test, I think it's nicer to just do "assertNotNull(flow)" - less code, and now the method doesn't need to declare that it throws a checked exception.